src/Interactive/EditorFeatures/Core/Extensibility/Interactive/CSharpVBResetCommand.cs

```diff
@@ -48,26 +50,28 @@ public IEnumerable<string> Names
 
         public string CommandLine
         {
-            get { return "[" + NoConfigParameterName + "]"; }
+            get { return "[" + NoConfigParameterName + "] [32|64]"; }
         }
 
         public IEnumerable<KeyValuePair<string, string>> ParametersDescription
         {
             get
             {
                 yield return new KeyValuePair<string, string>(NoConfigParameterName, InteractiveEditorFeaturesResources.Reset_to_a_clean_environment_only_mscorlib_referenced_do_not_run_initialization_script);
+                yield return new KeyValuePair<string, string>("32|64", $"Interactive host process bitness.");
             }
         }
 
         public Task<ExecutionResult> Execute(IInteractiveWindow window, string arguments)
         {
-            bool initialize;
-            if (!TryParseArguments(arguments, out initialize))
+            if (!TryParseArguments(arguments, out bool initialize, out bool? is64bit))
```

Comment:
>bool [](start = 71, length = 4)

I think that enum would be better for the case.

Reply:
How would that enum items be named?

Reply:
Bitness?

---
In reply to: [198254174](https://github.com/dotnet/roslyn/pull/28006#discussion_r198254174) [](ancestors = 198254174)

Reply:
The values?

Reply:
32
64
default? - if necessary

---
In reply to: [198254373](https://github.com/dotnet/roslyn/pull/28006#discussion_r198254373) [](ancestors = 198254373)

Reply:
32 is not a valid identifier

Reply:
enum Bitness
    {
        DefaultBitness,
        Bit32,
        Bit64
    }

---
In reply to: [198255961](https://github.com/dotnet/roslyn/pull/28006#discussion_r198255961) [](ancestors = 198255961)

Reply:
`Bit32` is odd. I have thought about this and didn't come up with any good names, so I used `bool?`. 
We need both {32, 64} and {32, 64, default} types.

Reply:
I see. Other then that, I do not have any comments. I still would prefer to reconsider "bool? Is64Bit" because it looks counterintuitive. 

You can decide but please re-consider it once again.

---
In reply to: [198258202](https://github.com/dotnet/roslyn/pull/28006#discussion_r198258202) [](ancestors = 198258202)